_docs_custom_metrics/generic-script/index.md

```diff
@@ -4,128 +4,174 @@ title: Generic script plugin
-| `period`                | Time interval in miliseconds where the agent polls the script for data.                                                                                  |
-|                         | The maximum value for this is 5000. Your script can only take 5 seconds to run or it will be killed by the agent.                                    |
-| `datatype`              | The data type of the created metric. Only `"DOUBLE"` is supported at this moment                                                                     |
-| `groups`                | The metric group that this metric should be added to. Groups inside other groups need to be separated by a `/`                                       |
-| `tags`                  | You can add a tag to add extra meaning to a metric (i.e. CPU_LOAD). Set as empty string to ignore this.
-| `attributeDescriptions` | The [attribute descriptions]({{ site.baseurl }}/events/custom-attributes/) of the event.                                                              |
-| `dimensions`            | The dimensions related to the given metric.                                                                                                          |
+| `id`                    | ID of the event.                                                                                                                                     |
+|                         | This ID will be used to match the data in the **data retrieval** model with the metric from the **configuration** mode.                              |
+|                         | IDs are local to the script and unrelated to the IDs given by the API.                                                                               |
+| `name`                  | The name of the event. (Note that this will be suffixed with '- Custom' to prevent collisions with other plugins.)                                   |
+| `description`           | Description of the event.                                                                                                                            |
+| `attributeDescriptions` | The [attribute descriptions]({{ site.baseurl }}/events/custom-attributes/) of the event.                                                             |
+
 
 ### Data retrieval mode
+
 Once the agent has been started, the Generic Script plugin will gather data once every minute by calling `<script> -d`.
 
-**Note:** The `<script> -d` should return within 5 seconds, otherwise it will be killed.
+{% include alert.html type="warning" text="Scripts executed by the generic script plugin are automatically stopped after 1 minute." %}
+
+Given the configuration above, the following output is an example of the script running in **data retrieval mode**:
+
+{% highlight bash %}
+M1 "1:eth0" 8.5
+M2 "2:dimension2_value1,3:dimension3_value1" 12345.6
+E1 -300 0 "The event triggered" "{"Exit code":137}" S
```

Comment:
"{"Exit code":137}"

Won't this give a person issue?

Reply:
Just tested it, it works.